Space Objects : Share options (#2390)
acterglobal/a3: .changes/2390-space-object-share.md

- [New] : Now you can directly share space objects (Pin, Event and TaskList) to new Boost from their detail page.

> - [New] : Now space objects can accessible by their own unique QR code.

Reviewer: ```suggestion
- [New] : You can now share any object via its own unique QR code.
```
